saleor/graphql/app/dataloaders.py

```diff
@@ -35,3 +38,49 @@ def batch_load(self, keys):
             extensions_map[extension.app_id].append(extension)
             app_extension_loader.prime(extension.id, extension)
         return [extensions_map.get(app_id, []) for app_id in keys]
+
+
+def get_app(raw_auth_token) -> Union[None, App]:
+    if raw_auth_token is None:
+        return None
+    tokens = AppToken.objects.filter(token_last_4=raw_auth_token[-4:]).values_list(
+        "app_id", "auth_token"
+    )
+    app_ids = [
+        app_id
+        for app_id, auth_token in tokens
+        if check_password(raw_auth_token, auth_token)
+    ]
+    return App.objects.filter(id__in=app_ids, is_active=True).first()
```

Comment:
This method shouldn't be in the data loader file, as it does not use any dataloader.

Reply:
I needed to refactor it out of its original position to mitigate circular dependency. Can you propose a proper location for it?

Reply:
IMO it is something more related to `saleor/app` than `saleor/graphql` module.

Reply:
I can refactor it back to original place (`saleor/graphql/context.py`) but it is getting removed anyway in next iteration because it's only used for lazy object.

Reply:
Maybe `saleor/app/utils.py`?

Reply:
Done.